releasenotes/notes/apm-profiling-upgrade-intakes-v2-1fbba441bbc08d20.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    upgrade profiling intake to v2 format, users can continue specifying
```

Comment:
We need to prefix all APM changelogs with `APM: `. And please start the sentence with a capital letter. Lastly, let’s remove the base URL only part because that’s not the usual behaviour.

Reply:
agreed